templates/init/debian/elasticsearch.j2

```diff
@@ -103,7 +103,7 @@ fi
 
 # Define other required variables
 PID_FILE="$PID_DIR/$NAME.pid"
-DAEMON=$ES_HOME/bin/elasticsearch
+DAEMON={{es_home}}/bin/elasticsearch
```

Comment:
With `{{es_home}}` it makes it consistent with how the [existing systemd](https://github.com/elastic/ansible-elasticsearch/blob/master/templates/systemd/elasticsearch.j2#L18) is handling it. I'm fine either way.